WRF_code/preprocess_hwe.py

```diff
@@ -72,15 +66,10 @@ def preprocess_stations(fn_lst):
     ccsm4 = preprocess_gcm(fn_lst[1])
     print("CCSM4 output preprocessed, time elapsed: " + str(round(time.clock() - tic, 2)) + "s")
 
-    def save_dfs(di, gcm):
-        for df in di:
-            fp = out_dir + gcm + "_" + df + ".csv" 
-            di[df].to_csv(fp, index=False)
-
     print("Saving files")
     tic = time.clock()
-    save_dfs(cm3, "CM3")
-    save_dfs(ccsm4, "CCSM4")
+    cm3.to_csv(out_dir + "CM3_hwe.csv")
+    ccsm4.to_csv(out_dir + "CCSM4_hwe.csv")
```

Comment:
Could these be appended into a single CSV before being saved?  If I'm reading the code right, there wouldn't be any collision because the model column would keep things distinct.

Reply:
Yeah definitely! I misunderstood - I thought the goal was one file for _each model_, my mistake!